### PR TITLE
unifying revisions to Purchases docs

### PR DESCRIPTION
**Pull request by christinalcole**
Closes ConvertKit/convertkit#10357

Changes:
- updated GET requests to query param syntax
- checked POST, PUT request to include required params
- swapped 'id' for 'purchase_id', to match specificity earlier in the docs
- removed "success" request details, to match rest of docs

To do:
- This is the only section that details failing requests.  I think this information is better suited for the "Getting started" section instead, so amended the issue for reviewing that section/will make the edit then (just to keep things straight/changes small)

QA:
- **Please** double-check that the PR is into `master`/is correct 😌 Opening the PR came with a lovely note to make sure it's not going to lord/slate; I think it's good now--but second eyes never hurt!
- Still need to check my commands via command-line, to make sure they work as advertised.  Based on the `API::V3::PurchasesController` code, should be good

### CODE REVIEW

File: source/includes/_purchases.md [hunk 1]
```diff
@@ -161,17 +155,17 @@ Show specific purchase by ID
 
 ### Endpoint
 
-GET /v3/purchases/#{id}
+GET /v3/purchases/#{purchase_id}
```
Comment (justrudd) [inline source/includes/_purchases.md]: You should put `api_secret` in the query string to be consistent with the other `GET` requests.

**Comment by justrudd** [comment]
This looks good. Not sure what `lord/slate` is but it looks like this will go to the correct place when merging.

File: source/includes/_purchases.md (hunk 1 above)
Comment (christinalcole) [reply source/includes/_purchases.md]: This section is only showing the endpoint/is the left side of the documentation, not an example request.  Going to leave it as-is for now, to see how everything looks post-merge.  If we want to change this format, we'll want to change it for all of the endpoint descriptions

**Comment by christinalcole** [comment]
verified GET changes work on command line; merging

**Pull request merged**